crates/wasm-encoder/src/core/code.rs

```diff
@@ -110,22 +110,25 @@ impl CodeSection {
     pub fn parse_section(
```

Comment:
Question for you, do you think it's worth keeping these methods? They're pretty easy to call as `RoundtripReencoder.parse_code_section(&mut code, section)` so I could see a case being made to removing them in favor of just having the `reencode.rs` implementation

Reply:
I put them in the initial commit to minimise the changed API. I think they could probably be removed (and I would remove the mut reference return). If using the RoundtripReencoder becomes common, everybody will need to also have the reencode trait in scope, but that should be a small cost to pay.